### ISSUE TITLE
[KEP-2400] [Swap] Add the ability to set a system-reserved quantity of swap on a node

### ISSUE THREAD

**Issue by iholder101** [kubernetes/kubernetes#120802]
### What would you like to be added?

Add the ability to set a system-reserved quantity of swap on a node.

Current work is being done here: https://github.com/kubernetes/kubernetes/pull/105271.

### Why is this needed?

KEP https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2400-node-swap

**Comment by pacoxu** [comment]
@iholder101 Is this issue still valid? If yes, I may give it a try again this release cyle.

**Comment by iholder101** [comment]
Hey @pacoxu!

Yes, I think it's still needed but is not one of the requirements for KEP-2400. That being said, I think it's still valuable for sure.

Please let me know if you have the bandwidth to implement this. If you won't perhaps I'll find the time for it.

**Comment by ajaysundark** [comment]
/retitle [Swap] Add the ability to set a system-reserved swap for node daemons.

Retitled based on earlier comment.

@iholder101 / @pacoxu If you're still interested in looking into this further, could you drive this with a separate KEP?

**Comment by iholder101** [comment]
> /retitle [Swap] Add the ability to set a system-reserved swap for node daemons.
> 
> Retitled based on earlier comment.
> 
> [@iholder101](https://github.com/iholder101) / [@pacoxu](https://github.com/pacoxu) If you're still interested in looking into this further, could you drive this with a separate KEP?

Yes, definitely. This should be considered for a separate KEP and is not in scope for KEP-2400.

**Comment by pacoxu** [comment]
/unassign 
as I may have no time on this recently

the previous try in https://github.com/kubernetes/kubernetes/pull/105271.
